src/pages/atomicdex/api/v20-dev/non_fungible_tokens/index.mdx

```diff
@@ -0,0 +1,577 @@
+# Non Fungible Tokens (NFTs)
+
+The AtomicDEX API supports [ERC1155](https://www.nftstandards.wtf/Standards/ERC1155+Multi+token) and [ERC721](https://www.nftstandards.wtf/Standards/ERC721+Non+Fungible+Standard) NFTs via the [Moralis API](https://docs.moralis.io/) on the Avalanche (AVAX), BNB Smart Chain (BNB), Ethereum (ETH), Fantom (FTM), Polygon (MATIC) networks.
+
+
+
+## NFT Information Methods
+
+- Get a list of your tokens with [get_nft_list](/atomicdex/api/v20-dev/non-fungible-tokens/get-nft-list)
+- Get a list of token transfers with [get_nft_transfers](/atomicdex/api/v20-dev/non-fungible-tokens/get-nft-transfers)
+- Get token metadata with [get_nft_metadata](/atomicdex/api/v20-dev/non-fungible-tokens/get-nft-metadata)
+
+
+## NFT Transaction Methods
+
+- Withdraw ERC721 tokens with [withdraw_nft](/atomicdex/api/v20-dev/non-fungible-tokens/withdraw-nft#erc721)
+- Withdraw ERC1155 tokens with [withdraw_nft](/atomicdex/api/v20-dev/non-fungible-tokens/withdraw-nft#erc1155)
+
+
+## Get a list of NFTs {{label : 'get_nft_list', tag : 'API-v2'}}
+
+
+### Request Parameters
+
+| Parameter     | Type             | Description                                                                                               |
+| ------------- | ---------------- | ----------------------------------------------------------------------------------------------------------|
+| chains        | array            | List of chains to scan for NFTs. Chains must be [activated](/atomicdex/api/legacy/coin_activation) first. |
+| url           | string           | URL link to [Moralis API](https://docs.moralis.io/) (or proxy).                                           |
```

Comment:
Moralis nft endpoint `https://deep-index.moralis.io/` wont work, bcz we removed moralis API-KEY owned by user from mm config.

Also, if user check https://docs.moralis.io link they wont understand what they need to find there and which moralis endpoint they should provide. In url we use proxy url `https://moralis-proxy.komodo.earth` instead of this part of full url for moralis requests.
For the future its better to provide link to moralis reference page directly https://docs.moralis.io/web3-data-api/evm/reference#get-nfts

![image](https://github.com/KomodoPlatform/komodo-docs-mdx/assets/52405288/d3d78bca-3ac0-4042-928f-48439561a0b8)

Reply:
Reworded, and added note about future potential param for user API key.